MSAL/MSAL.xcodeproj/project.pbxproj

```diff
@@ -8011,12 +8010,13 @@
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				CODE_SIGN_ENTITLEMENTS = test/app/mac/MSALMacTestApp.entitlements;
-				CODE_SIGN_IDENTITY = "Mac Developer";
+				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
 				COPY_PHASE_STRIP = NO;
 				DEBUG_INFORMATION_FORMAT = dwarf;
 				DEVELOPMENT_TEAM = UBF8T346G9;
+				ENABLE_HARDENED_RUNTIME = YES;
```

Comment:
Curious about this one too

Reply:
This is a Mac app security feature to prevent dynamic injection for libraries in Xcode

https://developer.apple.com/documentation/security/hardened-runtime

For apps communicating to Broker Xpc, this is required unless the app's bundle is putting into an exception list